packages/postgres-database/tests/conftest.py

```diff
@@ -208,15 +198,15 @@ async def connection(aiopg_engine: Engine) -> AsyncIterator[SAConnection]:
 @pytest.fixture
 async def asyncpg_engine(  # <-- WE SHOULD USE THIS ONE
     is_pdb_enabled: bool,
-    pg_sa_engine: sa.engine.Engine,
-    make_asyncpg_engine: Callable[[bool], AsyncEngine],
+    pg_sa_engine: sqlalchemy.engine.Engine,
```

Comment:
question: is there no way of running alembic in async mode? I have the feeling this usually runs faster. just a question, no need to do anything

Reply:
The migration scripts as sync ... so i would say that it needs to be executed in a thread or something similar